MIGRATION.md

```diff
@@ -629,13 +629,6 @@ export default {
 
 Alternatively, if you need to achieve a different behavior for a large number of files, you can provide a [custom indexer](https://storybook.js.org/docs/7.0/vue/configure/sidebar-and-urls#processing-custom-titles) to generate the titles dynamically.
 
-#### React v18 in the manager UI (including addons)
-
-Storybook 7 used React 16 in the manager. In Storybook 8 this is upgraded to react v18.
-Addons that inject UI into panels, tools, etc. are possibly affected by this.
-
-Addon authors are advised to upgrade to react v18.
-
```

Comment:
This was already documented further up: https://github.com/storybookjs/storybook/blob/jeppe/route-migration/MIGRATION.md#manager-addons-are-now-rendered-with-react-18

Reply:
Can you check whether there is any link or reference to the section that was removed?

Reply:
Good call. An org-wide search on the link didn't find any references.